### ISSUE TITLE
Run MusicVAE locally

### ISSUE THREAD

**Issue by InhyukYee** [tensorflow/magenta#1023]
Hello.

I am trying to run music_vae locally.

I followed the instructions as in music_vae README.md
When I try to run the script like as below.

```
$python music_vae_train.py --config=cat-mel_2bar_small --run_dir=/tmp/music_vae --mode=train --examples_path=/tmp/music_vae/mel_train_examples.tfrecord
```

It gives an error as below

```
Traceback (most recent call last):
  File "music_vae_train.py", line 332, in <module>
    console_entry_point()
  File "music_vae_train.py", line 328, in console_entry_point
    tf.app.run(main)
  File "/home/inhyuk/miniconda2/envs/magenta/lib/python2.7/site-packages/tensorflow/python/platform/app.py", line 48, in run
    _sys.exit(main(_sys.argv[:1] + flags_passthrough))
  File "music_vae_train.py", line 324, in main
    run(configs.CONFIG_MAP)
AttributeError: 'module' object has no attribute 'CONFIG_MAP'
```

I upgraded all the packages to the latest version.
I tested other models and it seems they are working fine.

I am trying to figure out why it can't import correctly.

Thanks,

**Comment by adarob** [comment]
Sorry for the delay. Were you able to solve this issue?

**Comment by ccassion** [comment]
I came across this issue myself yesterday. 
I was attempting to run the MusicVAE google colab code on my local machine.

I'm using Python 3.5 on MacOS High Sierra.

**Comment by InhyukYee** [comment]
No. I haven't solved yet.
but I am running music_vae in miniconda library now.

The path is like as below.
```
~/miniconda2/envs/magenta/lib/python2.7/site-packages/magenta/models/music_vae
```

I tried to run both of Ubuntu 16.04 and MacOS High Sierra, but they give same error.

**Comment by ccassion** [comment]
@InhyukYee My colleague was able to pinpoint my issue that may help you as well.

The files on git and colab *may be* different than pip.

So, **CONFIG_MAP** is actually **config_map** in configs the pip version of configs.py. It was changed on here: https://github.com/tensorflow/magenta/commit/b3e51fef5eb9f4875e12a9cb503e2e44d8d3c441#diff-9567b92253dc476296d67b51104b512e but I don't think the push was made to pip install version (?).

Anyhow, changing my configs call to (lower case) **config_map** worked for me.

Hope this resolves your issue and closes this.

**Comment by InhyukYee** [comment]
@ccassion Thanks for comment.
but It still gives an error.

If I run after changing **CONFIG_MAP** to **config_map**, it gives an error like below.
'''
Traceback (most recent call last):
  File "music_vae_train.py", line 327, in <module>
    console_entry_point()
  File "music_vae_train.py", line 323, in console_entry_point
    tf.app.run(main)
  File "/home/inhyuk/miniconda2/lib/python2.7/site-packages/tensorflow/python/platform/app.py", line 48, in run
    _sys.exit(main(_sys.argv[:1] + flags_passthrough))
  File "music_vae_train.py", line 319, in main
    run(configs.config_map)
  File "music_vae_train.py", line 300, in run
    task=FLAGS.task)
  File "music_vae_train.py", line 146, in train
    config.data_converter.is_training = True
AttributeError: 'Config' object has no attribute 'data_converter'
'''

**Comment by adarob** [comment]
Hello everyone! Apologies for the issues you are running into. The code is
still in a bit of flux as I am integrating several new features related to
a paper we recently submitted. I expect that things will be running
smoothly again

**Comment by adarob** [comment]
Oops, premature send there.

I expect things to be consistent between the colab and the GitHub repo by
the end of next week. Sorry for the inconvenience!

**Comment by adarob** [comment]
These issues should now be fixed with 0.3.6 (#1079).
